content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -372,6 +374,18 @@ implemented using the NetworkPolicy API.
 - The ability to prevent loopback or incoming host traffic (Pods cannot currently block localhost
   access, nor do they have the ability to block access from their resident node).
 
+## Recommendations for Users
+
+This section covers a set of recommended best practices to follow when using NetworkPolicies.
+
+- NetworkPolicies apply to connections and not packets. Thus if a NetworkPolicy is applied
+  in the middle of an existing connection its implementation defined as to whether the policy
```

Comment:
This is a non sequitur: NP _could_ have been defined in such a way that both (a) policies apply to connections and not packets, _and_ (b) the effect of changing policies when there are open connections _is_ well-defined. The fact that this situation is not well defined is not _because_ NPs apply to connections rather than packets. I would just omit the first sentence. (And the "Thus".)

Also, this isn't just about changing NetworkPolicies: the same ambiguity applies to when you change the Pods or Namespaces (eg, changing the labels on a Pod so that it starts/stops matching a NetworkPolicy). So something like: "When the set of NetworkPolicies that applies to an existing connection changes such that the connection would no longer be allowed, it is implementation defined as to whether the existing connection will be closed."

Reply:
ack I have tried to incorporate what you wrote here, hopefully the second update makes better sense :D